### PR TITLE
Add option to run with a plugin

### PR DESCRIPTION
**Pull request by salliewalecka**
By default, the plugin will not be used.  However, if you set the environmental variable INCLUDE_PLUGIN = Y and specify the parameter PLUGIN_SRC_DIR, the plugin will be copied over to plugins/external.

### CODE REVIEW

**Comment by arvindsv** [comment]
@rajiesh knows this best, I think.

File: lib/configuration.rb [hunk 1]
```diff
@@ -53,6 +53,14 @@ def agents_install_dir
       Pathname.new(env('AGENTS_INSTALL_DIR', '.')) + 'go-agents'
     end
 
+    def include_plugin
```
Comment (ketan) [inline lib/configuration.rb]: LGTM, mostly. Make this return a boolean instead and call it `def include_plugin?`

File: rakelib/server.rake [hunk 2]
```diff
@@ -23,6 +23,12 @@ namespace :server do
       f.extract_to("#{server_dir}")
     }
 
+    if setup.include_plugin.to_s == 'Y'
```
Comment (ketan) [inline rakelib/server.rake]: ` if setup.include_plugins?` reads better and is lesser cognitive load, IMO :)

**Comment by ketan** [comment]
I'm merging this, please submit any feedback in a different PR.

**Pull request merged**